config/config.go

```diff
@@ -21,6 +21,10 @@ type (
 		Repo          string `env-required:"true" yaml:"repo" env:"APP_REPO"`
 		Version       string `env-required:"true"`
 		EncryptionKey string `yaml:"encryption_key" env:"APP_ENCRYPTION_KEY"`
+		JWTKey        string `env-required:"true" yaml:"jwtKey" env:"APP_JWT_KEY"`
+		AuthEnabled   bool   `env-required:"true" yaml:"authEnabled" env:"APP_AUTH_ENABLED"`
```

Comment:
Setting `AuthEnabled: false` throws an error `Config error: field "AuthEnabled" is required but the value is not provided`

I think the error is caused by this tag `env-required:"true"`, as `false` might be treated as missing value.